npm-shrinkwrap.json

```diff
@@ -5982,6 +6017,11 @@
       "from": "sax@>=0.6.0",
       "resolved": "https://registry.npmjs.org/sax/-/sax-1.2.2.tgz"
     },
+    "seek-bzip": {
```

Comment:
As you can see from `udif.createReadStream()`, it is perfectly capable of streaming an image. The UDIF compresses each block range separately, so there's no need for a streaming bzip decompressor to stream an image.